CHANGELOG.md

```diff
@@ -18,6 +18,12 @@
 * [Breaking] Rename `yarpc.CreateThriftServiceFunc` to `yarpc.ServiceCreateFunc`
   as it is not thrift-specific.
 * Report version metrics for company-wide version usage information.
+* [Breaking] Rename functions inside `package dig` to match other APIs within UberFx:
+        * `Register` to `TryRegister`
```

Comment:
I haven't seen this `Try*` convention before, always thought it was implied by the fact that an error is returned.